docker-compose.lando-api.yml

```diff
@@ -11,15 +11,15 @@ services:
     image: suite_lando-api
     environment:
       PORT: 9000
-      VERSION_PATH: /version.json
+      VERSION_PATH: /config/version.json
```

Comment:
I don't think this is doing anything on lando-api as it defaults to "development" since the version file does not exist. What is the purpose of creating this volume to share across the containers?

Reply:
> I don't think this is doing anything on lando-api as it defaults to "development" since the version file does not exist. 

If you write the version config to `lando-api/docker/dev-config/version.json`, then that file will be mirrored at `/config/version.json`.

> What is the purpose of creating this volume to share across the containers?

The "shared `version.json`" logic existed in `docker-compose` here before my changes - I'm guessing that we want `lando-ui` and `lando-api` to have a synchronized `version.json` file.